addons/website_popup/views/website_popup.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <template id="assets_frontend" inherit_id="website.assets_frontend" name="Website Bounceexchange Theme">
+            <xpath expr="." position="inside">
+                <script type="text/javascript" src="/website_popup/static/src/js/website_popup.js"/>
+                <link rel='stylesheet' href='/website_popup/static/src/css/website_popup.css'/>
+            </xpath>
+        </template>
+
+        <template id="assets_editor" inherit_id="website.assets_editor" name="wesbite_popup Editor assets">
+            <xpath expr="/t" position="inside">
+                <script type="text/javascript" src="/website_popup/static/src/js/website.popup.editor.js" groups="base.group_website_publisher"></script>
+            </xpath>
+        </template>
+
+        <template id="website_popup_snippet" name="popup snippet" inherit_id="website.snippets">
+            <xpath expr="//div[@id='snippet_feature']" position="inside">
+                <div>
+                    <div class="oe_snippet_thumbnail">
+                        <img class="oe_snippet_thumbnail_img" src="/website_popup/static/src/img/bounce.png" />
+                        <span class="oe_snippet_thumbnail_title">Newsletter Popup</span>
+                    </div>
+
+                    <div class="banner_popup" data-list-id="0">
+                        <div class="alert alert-warning alert-dismissible fade in css_non_editable_mode_hidden o_not_editable" role="alert" contenteditable="false">
+                            <strong>Banner Popup! </strong>Banner popup snippet effect is active on this page. Click Here To <a id="edit_dialog" href="#"> Edit Dialog Content</a>
+                        </div>
+                        <div class="modal modal-dialog modal-md fade fade-custom" id="banner_popup" tabindex="-1" role="dialog">
+                            <div class="modal-content">
+                                <div class="modal-header o_not_editable" style="border: none;" contenteditable="false">

```

Comment:
please avoid inline style and if you don't want to display a title and border-bottom , then remove model-header div and move close button inside modal-body and make it editable false.